### PR TITLE
Travelling Salesman Problem in C

### PR DESCRIPTION
**Pull request by papri24majumder**
## 🛠️ Issue (Number)

Issue no #170 

## 👨‍💻 Changes proposed

Added traveling salesman problem in c language under dynamic programming

## ✔️ Check List (Check all the applicable boxes) 

- [✔️ ] My code follows the code style of this project.
- [ ✔️] This PR does not contain plagiarized content.
- [ ✔️] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
output

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,7 @@
 
 
 
+
```
Comment (zugzwang03) [inline README.md:73]: remove this change, as it is unnecessary

**Review by zugzwang03 — APPROVED** [review]

**Review by siddhi-244 — APPROVED** [review]
LGTM

**Comment by papri24majumder** [comment]
@siddhi-244 can I make another issue or wait till this pr gets merged?

**Comment by zugzwang03** [comment]
> @siddhi-244 can I make another issue or wait till this pr gets merged?

You can make another issue

**Comment by papri24majumder** [comment]
> 

ok thanks

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
Remove this updation to README.md from this PR

**Pull request closed without merging**